.gitignore

```diff
@@ -25,6 +25,7 @@ client/app/css/icons
 client/app/css/main.css
 client/app/css/main-scss.css
 client/app/css/main-scss.css.map
+package-lock.json
```

Comment:
Otherwise you could create a `.npmrc`:

```sh
$ cat .npmrc
package-lock=false
```

Reply:
Bonus: you can update as well the `.editorconfig` file.

`[{package,bower}.json]` => `[package.json]`

Reply:
I did that, thanks for the tip!